edit-post/components/header/more-menu/index.js

```diff
@@ -2,19 +2,18 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { IconButton, Dropdown } from '@wordpress/components';
+import { IconButton, Dropdown, MenuItemsGroup } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 import ModeSwitcher from '../mode-switcher';
 import FixedToolbarToggle from '../fixed-toolbar-toggle';
-import EditorActions from '../editor-actions';
 
-const element = (
+const MoreMenu = () => (
```

Comment:
Did this need to be changed to a function?

See also an explanation of this type of optimization: https://babeljs.io/docs/plugins/transform-react-constant-elements/

Reply:
> This transform _should be enabled only in production_ (e.g., just before minifying your code) because although it improves runtime performance, it makes warning messages more cryptic.

That's interesting. I haven't seen it before in the React docs. Can we enable the transform instead and leave all the low-level optimizations to Babel?